### ISSUE TITLE
[FEATURE REQUEST] Allow to show more/all beacon base blocks

### ISSUE THREAD

**Issue by Nycto97** [CERBON-MODS/Just-Enough-Beacons#3]
## Before submitting the issue

- [X] **I have checked that the requested feature is not a duplicate or, if it is, it needs to be reopened**
- ~~[] **I have checked that I am running the latest version of Better Totem Of Undying Mod**~~
- [X] *I have checked that I am running the latest version of Just Enough Beacons Mod*
- [X] **I have read the [contributing guidelines](https://github.com/CERBON-MODS/Just-Enough-Beacons/blob/master/CONTRIBUTING.md#feature-requests) and I agree with the [Code of Conduct](https://github.com/CERBON-MODS/Just-Enough-Beacons/blob/master/CODE_OF_CONDUCT.md)**

## Is your feature request related to a problem? Please describe.

Currently, the JEI screen only shows 28 blocks:

![2](https://github.com/CERBON-MODS/Just-Enough-Beacons/assets/83367226/0cdb9cc9-181f-4171-83ba-ecb25cb4fba3)

In my modpack, I currently have 156 blocks tagged with minecraft:beacon_base_blocks:

![1](https://github.com/CERBON-MODS/Just-Enough-Beacons/assets/83367226/32e3bd55-31e1-406b-b125-c58f859dade9)

Also, which blocks get shown, depends on what block you clicked to get to the JEI screen:

![3](https://github.com/CERBON-MODS/Just-Enough-Beacons/assets/83367226/51661fbb-efc9-4e8d-bb7d-75bb9d248057)

## Describe the solution you'd like

If possible, it would be nice to be able to see all beacon base blocks. A solution I can think of would be a scrollable list, although I don't know if that's possible inside a JEI screen.

An alternatie solution would be to make use of pages. Currently the JEB beacon base blocks JEI screen only uses 1 page:

![image](https://github.com/CERBON-MODS/Just-Enough-Beacons/assets/83367226/b0e27164-70f2-437b-9d8d-443929749a75)

**Comment by CerbonXD** [comment]
Actually, the mod already uses pages to show all the beacon base blocks and payments. Are you playing on 1.18 or 1.19+. It might be bugged for you for some reason.

**Comment by Nycto97** [comment]
We play on `Forge 1.19.2 43.3.5`
Don't know if the logs would help with anything, but providing them just in case.

Client:
[client-mod-list_2023-12-06_18.44.44.txt](https://github.com/CERBON-MODS/Just-Enough-Beacons/files/13593063/client-mod-list_2023-12-06_18.44.44.txt)
[latest.log](https://github.com/CERBON-MODS/Just-Enough-Beacons/files/13593075/latest.log)
[debug.log](https://github.com/CERBON-MODS/Just-Enough-Beacons/files/13593076/debug.log)


Server:
[server-mod-list_2023-12-06_18.48.33.txt](https://github.com/CERBON-MODS/Just-Enough-Beacons/files/13593065/server-mod-list_2023-12-06_18.48.33.txt)
[latest.log](https://github.com/CERBON-MODS/Just-Enough-Beacons/files/13593084/latest.log)
[debug.log](https://github.com/CERBON-MODS/Just-Enough-Beacons/files/13593086/debug.log)

**Comment by CerbonXD** [comment]
Are you playing a pack created on CurseForge or it's your own pack? If it's a pack on CurseForge please send me the name so I can install and test by myself. If it's your own pack, if you can, please send me a zip file with your mods and configs.

I see nothing wrong in the logs, I would like to test it myself.

**Comment by Nycto97** [comment]
My apologies for the delay.

Mods, configs, scripts,... changed a bit, but problem still persists.

I've put everything (hopefully :)) in a [zip archive](https://1drv.ms/u/s!AlY3YavyV2at1zoXjrm4d4UJpAKc?e=cpO5Au).

Let me know if you need anything else.

Thanks for your time and effort!

**Comment by CerbonXD** [comment]
I can't open the pack. It keeps giving me errors. Go to your CurseForge app and export the zip from there and send to me. I think it will work this way.

**Comment by Nycto97** [comment]
I don't have the CurseForge app. What I gave you are the folders you should manually put inside a minecraft instance. For this server I use the default launcher. It's not a modpack by definition.

I only have ATLauncher and that I only use to test out things.

I think if you manually take over the mods folder, it would already be enough to test. And if you test on singleplayer, make sure to combine the mods from client and server folder.

**Comment by CerbonXD** [comment]
https://github.com/CERBON-MODS/Just-Enough-Beacons/assets/84386993/17dc7c80-6e74-4f2d-b9e3-8f3be2397054

Sorry for the late reply, I was able to run the pack and for me the mod is working just fine as you can see in the video. Not sure why this is not working for you. Maybe reinstalling everything could help.

Btw, happy new year.

**Comment by Nycto97** [comment]
I'm lost for words, I'm so sorry... 2 days ago I checked and it still only showed me 1 page.

Now I check and I can see 6 pages... I have no clue how this occurred.

Thanks for your time and effort and sorry that this wasn't an issue apparently.

Thank you! Happy New Year for you too! ✨

**Edit as I'm typing this:**

I re-checked to be sure and I found the following:
- Pressing U or right-clicking the minecraft:beacon indeed shows all the pages
- Pressing U or right-clicking on a block that's tagged as applicable for beacon base will show only 1 page

Checking via beacon:

![image](https://github.com/CERBON-MODS/Just-Enough-Beacons/assets/83367226/30ab8cd3-e75d-4a8b-bd56-8cd344ce93b0)

Checking via blocks:

![image](https://github.com/CERBON-MODS/Just-Enough-Beacons/assets/83367226/36983044-1568-430a-ae30-5086a16a6afa)
